### ISSUE TITLE
Remove unrelated poster img and video links in custom_header_video.html in exampleSite

### ISSUE THREAD

**Issue by Daniel-The-Fox** [zjedi/hugo-scroll#192]
Hi there,

it looks like there are some obsolete file URLs in custom_header_video.html in exampleSite.
This is a bit irritating for new users to the Hugo scroll template (like me) imho. 😉

**Comment by zjedi** [comment]
yes, this could be related to recent changes related to i18n, I'll try having a look at that sometime this month.

**Comment by Daniel-The-Fox** [comment]
Ok great, thanks. 👍🏻
